.github/workflows/test.yml

```diff
@@ -9,8 +9,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, windows-2019, macos-11]
-        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        os: [ubuntu-20.04, windows-2019, macos-13, macos-15]
```

Comment:
Something of note is that we should be testing against Windows 2022 and Ubuntu 24.04 if we are upgrading macos versions as well